test/acceptance/service-fabrik-api.instances.director.spec.js

```diff
@@ -1432,6 +1423,134 @@ describe('service-fabrik-api', function () {
             });
         });
       });
+
+      describe('#schedule-update', function () {
+        it('should return 503 - schedule update feature not enabled', function () {
+          const mongourl = config.mongodb.url;
+          const mongoprovision = config.mongodb.provision;
+          delete config.mongodb.url;
+          delete config.mongodb.provision;
+          mocks.uaa.tokenKey();
+          mocks.cloudController.getServiceInstance(instance_id, {
+            space_guid: space_guid,
+            service_plan_guid: plan_guid
+          });
+          mocks.cloudController.findServicePlan(instance_id, plan_id);
+          mocks.cloudController.getSpaceDevelopers(space_guid);
+          return chai.request(apps.external)
+            .put(`${base_url}/service_instances/${instance_id}/schedule_update`)
+            .set('Authorization', authHeader)
+            .set('Accept', 'application/json')
+            .send({
+              type: 'online',
+              repeatInterval: '*/1 * * * *'
+            })
+            .catch(err => err.response)
+            .then(res => {
+              expect(res).to.have.status(503);
+              expect(res.body.message).to.eql(`${CONST.FEATURE.SCHEDULED_UPDATE} feature not enabled`);
+              config.mongodb.url = mongourl;
+              config.mongodb.provision = mongoprovision;
+              mocks.verify();
+            });
+        });
+
+        it('should return 400 - Badrequest on skipping mandatory params', function () {
+          mocks.uaa.tokenKey();
+          mocks.cloudController.getServiceInstance(instance_id, {
+            space_guid: space_guid,
+            service_plan_guid: plan_guid
+          });
+          mocks.cloudController.findServicePlan(instance_id, plan_id);
+          mocks.cloudController.getSpaceDevelopers(space_guid);
+          return chai.request(apps.external)
+            .put(`${base_url}/service_instances/${instance_id}/schedule_update`)
+            .set('Authorization', authHeader)
+            .send({})
+            .catch(err => err.response)
+            .then(res => {
+              expect(res).to.have.status(400);
+              expect(res.body).to.eql({});
+              mocks.verify();
+            });
+        });
+
+        it('should return 201 OK', function () {
```

Comment:
in description you can put the use case.

Reply:
describe section mentions the use case. This is one of the conditions in that use case. [This means successfully able to schedule update]